packages/fast-components-class-name-contracts-msft/src/action-toggle/index.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * The class name contract for the action toggle component
+ */
+export interface ActionToggleClassNameContract {
+    /**
+     * The root of the action toggle component
+     */
+    actionToggle: string;
```

Comment:
I believe this should be optional, all class names should be optional.

Reply:
I thought we decided the opposite? In a styled component, the base class name should always be there. That is what makes it styled.

@nicholasrice I feel like we have been around about this more than once...

Reply:
I think Jane is correct here, all class names should be optional. In a previous PR I think this was incorrectly implied.

Reply:
Can we have a formal statement on why this is the case, so we don't run into this back-and-forth again?

Reply:
Maybe a good topic to note in our contributing documentation if we are getting confused.

Reply:
Made optional for this and action-trigger as well (it was also incorrect)